blender-movie-exercise/js/index.js

```diff
@@ -0,0 +1,100 @@
+/* Importing the json files into the javascript file. */
+import videoJsonContent from "/blender-movie-exercise/js/video.json" assert { type: "json" };
+import posterJsonContent from "/blender-movie-exercise/js/posters.json" assert { type: "json" };
+
+// console.log(videoJsonContent);
+// console.log(posterJsonContent);
+
+let movieDisplayContainer = document.getElementById("movieDisplayContainer");
```

Comment:
Can we use const here and check if still works for all the document selected variables?